package.json

```diff
@@ -99,7 +99,7 @@
     "ember-cli-browserstack": "^1.1.0",
     "ember-cli-content-security-policy": "^2.0.0-4",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-fastboot": "^2.2.3",
+    "ember-cli-fastboot": "https://gitpkg.now.sh/ember-fastboot/ember-cli-fastboot/packages/ember-cli-fastboot?master",
```

Comment:
This let's us use https://github.com/ember-fastboot/ember-cli-fastboot/pull/820, which fixes a broken import of `deprecate` that breaks canary, but is not released yet. Should be updated when a stable v3 is published, hopefully soon: https://discord.com/channels/480462759797063690/485861149821239298/870288184259510344